src/app/containers/MediaPlayer/__snapshots__/index.test.jsx.snap

```diff
@@ -51,4 +51,4 @@ exports[`MediaPlayer Calls the correct props Calls the canonical player when pla
 
 exports[`MediaPlayer Fails and returns early when component is toggled off 1`] = `null`;
 
-exports[`MediaPlayer Fails and returns early when there is no versionId 1`] = `null`;
+exports[`MediaPlayer Fails and returns early when there is no versionId 1`] = `null`;
```

Comment:
I don't think this file needs committing.

`git checkout latest -- src/app/containers/MediaPlayer/__snapshots__/index.test.jsx.snap` will reset the file to latest